source/_data/plugins.yml

```diff
@@ -24,6 +24,11 @@
       description: Watches and bundles your spec files via webpack.
       link: https://github.com/cypress-io/cypress-webpack-preprocessor
       keywords: [webpack]
+      
+    - name: cypress-testrail-reporter
```

Comment:
Placing this here displays the `cypress-testrail-reporter` as under the `Preprocessors`, which isn't technically accurate. Maybe it should go under Development Tools???? It's a bit more of a category for "Reporting" than anything - but we don't have that category, should we make that category??

Reply:
Okay. 🤔I guess that would/could be it's own category. Would it be okay hanging out by itself for a little while?